src/index.json

```diff
@@ -1046,8 +1046,8 @@
         ],
         "dms-preview": [
             {
-                "downloadUrl": "https://azcliorcas.blob.core.windows.net/azclipath/dms_preview-0.9.0-py2.py3-none-any.whl",
-                "filename": "dms_preview-0.9.0-py2.py3-none-any.whl",
+                "downloadUrl": "https://azcliorcas.blob.core.windows.net/azclipath/dms_preview-0.10.0-py2.py3-none-any.whl",
```

Comment:
You can append the new version information to dms-preview list.

Reply:
We were explicitly told to overwrite. Additionally, at a glance I don't see any other extension with multiple version entries  in here.